uptane-standard.md

```diff
@@ -656,6 +656,9 @@ A primary downloads, verifies, and distributes the latest time, metadata and ima
 1. Send metadata to secondaries ({{send_metadata_primary}})
 1. Send images to secondaries ({{send_images_primary}})
 
+
+Please note, the subsequent sections concerning requirements for a primary, does not prohibit implementing primary-capability on an ECU which does not communicate directly with the Uptane repositories. This allows for implementations to have multiple ECUs within the vehicle performing functions equivalent to a primary.
```

Comment:
This should also note that each ECU shall have a single primary responsible for its updates even if there are multiple primaries.

Reply:
Clarified in subsequent push based on feedback.